server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -281,7 +281,9 @@ public Address getAddress() {
    * the correction process.
    */
```

Comment:
(blocking) Please update the javadoc to reflect the new behavior.

Reply:
Done thanks for catching!